### PR TITLE
Quicker event access

### PR DESCRIPTION
**Pull request by dneise**
I propose to make event access easier.

This is  digicampipe, so the default case for all of us is using the digicam. The `event_source` should know about this default and act accordingly. 

People can still create their own camera config files, and create their own camera, like:
```python
my_cam = utils.Camera('my_config_file.txt')
for event in event_stream('event_file.fits.fz', camera=my_cam):
    pass
```

But the default is to use `utils.DigiCam`.

This should make the life of normal people easier, while camera specialists, who want to use several different cameras for testing, need to write a bit more code, but still can do their job.

-----

In addition: I added one "nice-to-have" feature, which is you can either give a list of paths, or a single path, and both works... I so often made the mistake of giving a single file only ... I thought, maybe others do to.

For a comparison of the way it was before, and the way it is now, see: 
https://github.com/cta-sst-1m/digicampipe/blob/nice_event_source_and_pep8/digicampipe/tests/test_event_stream.py

### CODE REVIEW

File: digicampipe/io/event_stream.py [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
-
+def event_stream(
+    filelist_or_url,
+    camera=None,
+    camera_geometry=None,
+    expert_mode=None,
+    max_events=None,
+    mc=False
+):
+    if isinstance(filelist_or_url, (str, bytes)):
```
Comment (calispac) [inline digicampipe/io/event_stream.py]: FYI there is something similar for numpy array and single values : 

https://docs.scipy.org/doc/numpy/reference/generated/numpy.atleast_1d.html

I was quickly looking if there is something similar for strings and lists of strings but could not find it...

Comment (dneise) [reply digicampipe/io/event_stream.py]: Yes, this is an ugly point, it makes the interface "unclear". 
A function should have a clear interface. 

But, I thought this function is very exposed to the "end-user" so adding this feature felt "okay-ish" ... 

I'm not sure anymore though... maybe its fine

Comment (calispac) [reply digicampipe/io/event_stream.py]: Yes maybe a bit "hard" to read but I suppose a comment would make it easier

Comment (calispac) [reply digicampipe/io/event_stream.py]: also you can overwrite the variables right? 

just removing the else statement and: 

```python
def event_stream(
    filelist,
   ....
   )
...
   if isinstance(filelist, (str, bytes)):
      filelist = [filelist]

...
```

Comment (dneise) [reply digicampipe/io/event_stream.py]: :-D ... I guess you know my opinion about comments. When they are needed, the interface is broken :-D

**Comment by dneise** [comment]
thanks

**Pull request merged**